src/generic_ob2ioda.py

```diff
@@ -1,70 +0,0 @@
-#!/usr/bin/python
```

Comment:
@BenjaminRuston  Did you mean to delete this?

Reply:
yup